Source/Source.php

```diff
@@ -141,7 +141,7 @@ function menu_main() {
 		$t_menu_options = array();
 
 		if ( plugin_config_get( 'show_repo_link' ) ) {
-			$t_page = plugin_page( 'index', false, 'Source' );
+			$t_page = plugin_page( 'index', true, 'Source' );
```

Comment:
This was actually fixed in 1491ba13b288d1b8ac18d3025e3db29e62265cb5, see also http://www.mantisbt.org/bugs/view.php?id=22107